osu.Game/Overlays/Toolbar/Toolbar.cs

```diff
@@ -24,6 +24,10 @@ public class Toolbar : OverlayContainer
         private ToolbarModeSelector modeSelector;
         private ToolbarUserArea userArea;
 
+        protected override bool HideOnEscape => false;

```

Comment:
All these overrides make me think we actually might want these to just be boolean members instead of overridable properties.